backend/src/email/middlewares/email-transactional.middleware.ts

```diff
@@ -232,7 +304,7 @@ export const InitEmailTransactionalMiddleware = (
     })
     res.status(200).json({
       has_more: hasMore,
-      data: messages.map(convertMessageModelToResponse),
+      data: messages.map((m) => convertMessageModelToResponse(m)),
```

Comment:
not sure if there're any TS flagging but we don't need to do this as the current way is already just passing the first arg already

Reply:
it complains when I change it back though, probably cause convertMessageModelToResponse takes in 2 args now

Reply:
Will checkout in my local and have a look

Reply:
ah forgot that map function will have a bunch of other params as well